modules/biobank/jsx/specimenTab.js

```diff
@@ -322,7 +323,7 @@ class SpecimenTab extends Component {
         name: 'containerBarcode',
         type: 'text',
       }},
-      {label: 'Coordinate', show: true},
+      {label: t('Coordinate', {ns: 'loris'}), show: true},
```

Comment:
I don't think this is used anywhere els in loris? Should probably be in the biobank namespace